lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -219,4 +223,5 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         "language": video_descriptor.get_default_transcript_language(),
         "category": video_descriptor.category,
         "id": unicode(video_descriptor.scope_ids.usage_id),
+        "encoded_videos": val_video_info
```

Comment:
What are all the fields in `val_video_info`?  Some fields may be private to the server and not sent to the client.  

Specifically, for now, only the following fields should be included: `url`, `file_size`.
Otherwise, the following fields would be sent: `created`, `modified`, `id`.